config/config.proto

```diff
@@ -82,3 +83,7 @@ message ConfigUnion {
     ZoneConfig zone = 1;
   }
 }
+
+message SystemConfig {
+  repeated proto.KeyValue values = 1 [(gogoproto.nullable) = false];
```

Comment:
you could possibly use a `map` here, but i guess you'll see when you start using this stuff